### PR TITLE
Tobbe/about us

### PR DESCRIPTION
**Pull request by TobiasKjernell**
merged main into about us branch, fixed some conflicts and sorted code and files into seperated folders

### CODE REVIEW

File: html/about-us.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                                <div class="card__side card__side--back card--uganda">
+                                    <div class="card__flag--uganda"></div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="grid__item">
+                            <div class="card">
+                                <div class="card__side card__side--front">
+                                    <div class="card__picture card__picture--sweden">
+                                        &nbsp;
```
Comment (VaishaliGovindaraj) [inline html/about-us.html:204]: can you explain this

File: html/about-us.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                            <span>About us</span>
+                        </a>
+                    </li>
+                </ul>
+                <div class="legal">
+                    &copy; 2024 by Git-Project
+                </div>
+            </div>
+            <div class="main-content">
+                <div class=".about-us">
```
Comment (Svpriyaa2808) [inline html/about-us.html:88]: can we define class name with dot infront of name

File: html/about-us.html (hunk 1 above)
Comment (TobiasKjernell) [reply html/about-us.html:204]: &nbsp; is a html typo for generate a "space". i used it to just give some info on the card picture section to trigger events.

File: html/about-us.html (hunk 2 above)
Comment (TobiasKjernell) [reply html/about-us.html:88]: good find. that was a miss from me.

File: html/about-us.html [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                        We are five new students currently studying Frontend at Futuregames in Stockholm, Sweden. Our
+                        first weeks at Futuregames we learned about HTML & CSS and right now we are battling with the
+                        essentials of GIT in a group project.
+                    </div>
+                    <div class="about-us__grid">
+                        <div class="grid__item">
+                            <div class="card">
+                                <div class="card__side card__side--front">
+                                    <div class="card__picture card__picture--india-1">
+                                        &nbsp;
```
Comment (Svpriyaa2808) [inline html/about-us.html:99]: can you explain this

**Review by andrewkalumba — COMMENTED** [review]
All good

File: styles/card.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+.card:hover .card__side--back {
+    transform: rotateY(0);
+}
+
+.card--india {
+    background-image: linear-gradient(to bottom, #ffb900, #fff, green);
+}
+
+.card--uganda {
```
Comment (VaishaliGovindaraj) [inline styles/card.css:47]: should  we be  using variable here?

Comment (TobiasKjernell) [reply styles/card.css:47]: since i use this once, we don't have to.

**Review by andrewkalumba — APPROVED** [review]

**Review by Svpriyaa2808 — APPROVED** [review]
Looks good

**Review by VaishaliGovindaraj — APPROVED** [review]

**Review by Svpriyaa2808 — APPROVED** [review]

**Pull request merged**